index.html

```diff
@@ -0,0 +1,174 @@
+      <p style="color: brown;">BYE TOUGH DAY</p>
+    </div>
+  </div>
+
+  <!-- form sign up -->
+  <form class="form-register" id="form-sign-up">
+    <div class="form-group1">
+      <label for="first_name">First Name</label>
+      <input type="text" class="form-control" id="first_name">
+    </div>
+    <div class="form-group1">
+      <label for="last_name">Last Name</label>
+      <input type="text" class="form-control" id="last_name">
+    </div>
+    <div class="form-group1">
+      <label for="email">Email</label>
+      <input type="email" class="form-control" id="email">
+    </div>
+    <div class="form-group1">
+      <label for="password">Password</label>
+      <input type="password" class="form-control" id="password">
+    </div>
+    <div class="form-group1">
+      <label for="date_of_birth">Date Of Birth</label>
+      <input type="date" class="form-control" id="date_of_birth">
+    </div>
+    <button id='btn-sign' type="submit" class="btn btn-primary ">Sign Up</button>
+  </form>
+
+  <!-- form login -->
+  <div id='div-login'>
+    <form id="formContent" class="form-login   w3-animate-top">
+      <div class="form-group">
+        <label for="emailLogin">Email</label>
+        <input type="email" class="form-control" id="emailLogin">
+      </div>
+      <div class="form-group">
+        <label for="passwordLogin">Password</label>
+        <input type="password" class="form-control" id="passwordLogin">
+      </div>
+      <button type="submit" class="btn btn-primary">Log In</button><br><br>
+      or <br><br>
+      <div class="g-signin2" data-onsuccess="onSignIn"></div>
+    </form>
+  </div>
+
+  <!-- All Todo List -->
+  <div class="todos">
+    <div class="jumbotron">
+      <h1 class="display-4">My TodoList</h1><br>
+      <h4>what are you gonna do today?</h4><br>
+      <!-- form adding -->
+      <form id="formAdd" class='addTodoForm'>
+        <label for="title"></label>
+        <input type="text" id="title" placeholder="title" class="addInput"><br><br>
+        <label for="description"></label>
+        <input type="text" id="description" placeholder="description" class="addInput"><br><br>
+        <label for="due_date"></label>
+        <input type="date" id="due_date" placeholder="due_date" class="addInput"><br><br><br>
+        <input style="background-color:#05dfd7" type="submit" value="add todo">
+      </form>
+      <hr>
+      <!-- form editing -->
+      <form class='editTodoForm'>
+        <input type="hidden" id="editId"><br><br>
+        <label for="editTitle"></label>
+        <input type="text" id="editTitle" class="addInput"><br><br>
+        <label for="editDescription"></label>
+        <input type="text" id="editDescription" class="addInput"><br><br>
+        <label for="editDue_date"></label>
+        <input type="date" id="editDue_date" class="addInput"><br><br><br>
+        <label for="editStatus"></label>
+        <input type="text" id="editStatus" class="addInput"><br><br><br>
```

Comment:
ini text ya, sebaiknya kalau dibikin text, mungkin dipojokan atas atau bawah kolomnya ada note kecil warna samar gitu, misal "*fill it with true/false" jadi user ga kehilangan jejak :) ini kaitannya dengan UX ya, btw nice try!